docs/configuration/integrations/cadvisor-config.md

```diff
@@ -0,0 +1,115 @@
+
+The cAdvisor integration requires some broad privileged permissions to the host. Without these permissions the metrics will not be accessible. This means that the agent must *also* have those elevated permissions.
+
+A good example of the required file, and system permissions can be found in the docker run command published in the [cAdvisor docs](https://github.com/google/cadvisor#quick-start-running-cadvisor-in-a-docker-container).
+
+Full reference of options:
+
+```yaml
+  # Enables the cadvisor integration, allowing the Agent to automatically
+  # collect metrics for the specified github objects.
+  [enabled: <boolean> | default = false]
+
+  # Sets an explicit value for the instance label when the integration is
+  # self-scraped. Overrides inferred values.
+  #
+  # The default value for this integration is inferred from the hostname portion
+  # of api_url.
+  [instance: <string>]
+
+  # Automatically collect metrics from this integration. If disabled,
+  # the cadvisor integration will be run but not scraped and thus not
+  # remote-written. Metrics for the integration will be exposed at
+  # /integrations/cadvisor/metrics and can be scraped by an external
+  # process.
+  [scrape_integration: <boolean> | default = <integrations_config.scrape_integrations>]
+
+  # How often should the metrics be collected? Defaults to
+  # prometheus.global.scrape_interval.
+  [scrape_interval: <duration> | default = <global_config.scrape_interval>]
+
+  # The timeout before considering the scrape a failure. Defaults to
+  # prometheus.global.scrape_timeout.
+  [scrape_timeout: <duration> | default = <global_config.scrape_timeout>]
+
+  # Allows for relabeling labels on the target.
+  relabel_configs:
+    [- <relabel_config> ... ]
+
+  # Relabel metrics coming from the integration, allowing to drop series
+  # from the integration that you don't care about.
+  metric_relabel_configs:
+    [ - <relabel_config> ... ]
+
+  # How frequent to truncate the WAL for this integration.
+  [wal_truncate_frequency: <duration> | default = "60m"]
+
+  #
+  # cAdvisor-specific configuration options
+  #
+
+  # Convert container labels and environment variables into labels on prometheus metrics for each container. If false, then only metrics exported are container name, first alias, and image name.
+  [store_container_labels: <boolean> | default = true]
+
+  # List of container labels to be converted to labels on prometheus metrics for each container. store_container_labels must be set to false for this to take effect.
+  white_listed_container_labels:
+    [ - <string> ]
+
+  # List of environment variable keys matched with specified prefix that needs to be collected for containers, only support containerd and docker runtime for now.
+  env_metadata_whitelist:
+    [ - <string> ]
+
+  # List of cgroup path prefix that needs to be collected even when docker_only is specified.
+  raw_cgroup_prefix_whitelist:
+    [ - <string> ]
```

Comment:
nit: Can we have these be allowlist?

Reply:
Yes.. But.. That would make the terminology different than the upstream project. I'm *certain* that the meaning won't be lost, but I dunno if this might be confusing to those who are used to using cAdvisor directly.🤔

Reply:
We can reference the original flag names in the comment, if it helps

Reply:
Meh, I'm good changing this all to allowlist, and waiting to see if people complain ;)